1-exercises/B-while-loop/exercise.js

```diff
@@ -6,7 +6,16 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+  // TODO
+  let i = 1;
+  let string = "0";
+  while (i < n * 2) {
+    if (i % 2 === 0) {
+      string += `,${i}`;
+    }
+    i++;
+  }
+  console.log(string);
```

Comment:
Nice as this functions works correctly. Perhaps we could simplify the logic a bit. 
Given n = 0, 1, 2 etc and n * 2 = 0, 2, 4. Could you derive another solution to this problem?